src/group/mls_group/create_commit.rs

```diff
@@ -168,7 +168,8 @@ impl MlsGroup {
                     let dirpath = treemath::direct_path_root(
                         provisional_tree.get_own_node_index(),
                         provisional_tree.leaf_count(),
-                    );
+                    )
+                    .expect("create_commit_internal: TreeMath error when computing direct path.");
```

Comment:
You added `expect` everywhere here. This implies that these errors must not happen because they result in panics. I'm not sure that this is really what we want. AFAIK treemath might return an error when there are inconsistencies in the tree. Couldn't this be recoverable or a test on an incoming tree?

Reply:
I thought so, too, initially. But Raphael pointed out that TreeMath really only operates on the array-representation of the tree, so if we try to, for example, get the child of a leaf, it must be an implementation error, because that will be the case for the representation of any tree, regardless if the content of the nodes make sense.

Reply:
Hm, not so sure if this is the best way. But we'll have to tackle error handling again anyway (#83) let's revisit then.